### PR TITLE
ticket_controller rework

### PR DESCRIPTION
**Pull request by KMosc**
Changelog

- [x] add Gems => [Rspec, sentry]
- [x] Controller rework

### CODE REVIEW

**Comment by KMosc** [comment]
@wmaciejak new changes.

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '3.0.1'
 
 gem 'rspec', '~> 3.0'
```
Comment (wmaciejak) [inline Gemfile:6]: It's not a part of this PR, but `rspec` should be move to group `developement and test`

File: app/controllers/ticket_controller.rb [hunk 2]
```diff
@@ -2,25 +2,28 @@ class TicketController < ApplicationController
...
-
     @tickets = Ticket.where(ticket_desk_id: params[:ticket_desk_id]).where(cinema_hall_id: params[:cinema_hall_id])    
     render json: @tickets
   end
 
   def show
     render json:[]
   end
+  
+  def new
```
Comment (wmaciejak) [inline app/controllers/ticket_controller.rb:13]: `new` action is not needed, cause it's related to Rails views. Please work only with `create` action

File: app/controllers/ticket_controller.rb [hunk 3]
```diff
@@ -2,25 +2,28 @@ class TicketController < ApplicationController
...
+  end
 
   def create
     #Cant buy if there is not such ticketdesk or cinemahall
     #Calling multiple methods using .send and []
       if ticket_available?
         if Ticket.exists?(:paid => false)  
-          Ticket.where(:paid => false).update_all(paid: true)
-          render :json => "You've paid for ticket that You had booked."
+          confirm_reservation()
```
Comment (wmaciejak) [inline app/controllers/ticket_controller.rb:22]: When there is no arguments, you can execute method without bracket - `confirm_reservation`

File: config/routes.rb [hunk 4]
```diff
@@ -9,9 +9,8 @@
 end
 
 scope 'ticket' do
-  post 'ticket', to: 'ticket#create', as: :buy
-
-  post 'ticket', to: 'ticket#booked'
+  post "ticket", to: 'ticket#create', as: 'ticket_create'
+  post "ticket", to: 'ticket#bookin', as: 'ticket_bookin'
```
Comment (wmaciejak) [inline config/routes.rb:14]: In creating standard REST API we very rarely changing name of methods used in controllers. Please consider how to make API consistent by using only `index/create/update/delete` methods.

**Comment by wmaciejak** [comment]
@KMosc remember to not merge before code review will pass, so reviewer will accept the pull request :)

**Pull request merged**